### PR TITLE
Introduce a nightly_releaser variable

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ehelms** [comment]
What solution does variablizing it solve to help with context?

**Comment by ekohl** [comment]
I thought it could help in the Jenkins job but there's probably no easy way to use that variable in an override. https://github.com/theforeman/obal/pull/110 is probably a better solution.

File: package_manifest.yaml [hunk 1]
```diff
@@ -343,7 +345,7 @@ katello_packages:
     rubygem-hammer_cli_katello: {}
     rubygem-katello:
       releasers:
-        - koji-katello-jenkins
+        - "{{ nightly_releaser }}"
```
Comment (komidore64) [inline package_manifest.yaml]: @ehelms correct me if I'm wrong, but isn't this going to break releasing Katello? Even for actual Katello releases, it gets the source from the last successful Jenkins nightly build.

Comment (ekohl) [reply package_manifest.yaml]: It shouldn't. This change should be a noop but when https://github.com/theforeman/obal/pull/110 is merged we also need to set `nightly_package_tito_releaser_args`. I can update the PR so they're compatible.

Thinking about it, I'm wondering if we need a `release_is_nightly` parameter for jobs that are only nightly. Then after branching we can flip the boolean when we add the actual released sources.

Comment (komidore64) [reply package_manifest.yaml]: theforeman/obal#110 has been merged.

**Comment by ekohl** [comment]
Updated. I think this config now matches reality. With this Jenkins can be modified to `obal nightly` and with the args for job name and job id. It allows still releasing hammer with the included gems while releasing nightlies.

**Pull request merged**